packages/firebase_ml_vision/lib/src/text_recognizer.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
This is only a file rename `text_detector.dart` -> `text_recognizer.dart` with accompanied class rename